migrations/master/README.md

```diff
@@ -61,3 +61,78 @@ services:
 ```
 
 You might need to reuse or adapt those changes for your setup.
+
+
+## cas
+From 22.x to 23.x
+To activate password rotation policy for ldap, user need to run these commands:
+
+```
+ldapmodify -H ldap://localhost:389 -D cn=admin,dc=georchestra,dc=org -w secret
+dn: cn=module{0},cn=config
+changetype: modify
+add: olcModuleLoad
+olcModuleLoad: ppolicy.la
+```
+```
+ldapadd -D cn=admin,dc=georchestra,dc=org -w secret  -f /etc/ldap/schema/ppolicy.ldif  (inside ldap docker container if docker was used)
+
+```
+```
+ldapadd -H ldap://localhost:389 -D cn=admin,dc=georchestra,dc=org -w secret
+dn: olcOverlay=ppolicy,olcDatabase={1}mdb,cn=config
+objectClass: olcOverlayConfig
+objectClass: olcPPolicyConfig
+olcOverlay: ppolicy
+olcPPolicyDefault: cn=default,ou=pwpolicy,dc=georchestra,dc=org
+olcPPolicyHashCleartext: FALSE
+olcPPolicyForwardUpdates: FALSE
+olcPPolicyUseLockout: FALSE
+```
+```
+ldapadd -H ldap://localhost:389 -D cn=admin,dc=georchestra,dc=org -w secret
+dn: ou=pwpolicy,dc=georchestra,dc=org
+objectClass: organizationalUnit
+objectClass: top
+ou: pwpolicy
+
+dn: cn=default,ou=pwpolicy,dc=georchestra,dc=org
+objectClass: person
+objectClass: pwdPolicyChecker
+objectClass: pwdPolicy
+cn: pwpolicy
+sn: pwpolicy
+pwdAttribute: userPassword
+pwdMinAge: 0
+pwdMaxAge: 31536000
+pwdGraceAuthnLimit: 0
+pwdExpireWarning: 2592000
+```
+```
+ldapadd -H ldap://localhost:389 -D cn=admin,dc=georchestra,dc=org -w secret
+dn: cn=pwd-no-expire,ou=pwpolicy,dc=georchestra,dc=org
+objectClass: person
+objectClass: pwdPolicyChecker
+objectClass: pwdPolicy
+cn: pwpolicy
+cn: pwd-no-expire
+sn: pwpolicy
+pwdAttribute: userPassword
+pwdMinAge: 0
+pwdMaxAge: 0
+```
+```
+ldapmodify -H ldap://localhost:389 -D cn=admin,dc=georchestra,dc=org -w secret
```

Comment:
useless, since was already done from 22.0 to 23.0

Reply:
Here, I'm changing policy to no expire policy so these users will not be included in rotation password policy. I think, this is necessary @fvanderbiest.

Reply:
You're 100% right !
Sorry Marwane. I was too quick in my review